spec/features/collection_drafts/publish_collection_draft_spec.rb

```diff
@@ -218,6 +218,7 @@
       draft = create(:full_collection_draft, user: User.where(urs_uid: 'testuser').first, native_id: 'not & url, encoded / native id', draft_short_name: 'test short name')
       visit collection_draft_path(draft)
       click_on 'Publish'
+      sleep(1)
```

Comment:
Why does this one need a sleep but, the `when publishing a collection draft record` suggest add comment?

Reply:
Next round of changes I'm going to remove the sleep, as it didn't make any difference.